app/models/middleware_server_performance.rb

```diff
@@ -0,0 +1,69 @@
+class MiddlewareServerPerformance < ActsAsArModel
+  set_columns_hash(
+    :id            => :string,
+    :name          => :string,
+    :resource_type => :string,
+    :feed          => :string,
+    :ems_ref       => :string,
+    :start_time    => :datetime,
+    :end_time      => :datetime
+  )
+
+  alias resource_id id
+  alias resource_name name
+
+  def self.build_results_for_report_middleware(options)
+    start_time, end_time, interval = parse_time_interval(options)
+    results = []
+    MiddlewareServer.find_each do |ms|
```

Comment:
Change the name makes sense where there is a clear misunderstanding or it is confusing.
Long names are good when the inner block has some complex logic.
In this case I prefer to not change this one.
In ManageIQ there are clear examples of same style:
https://github.com/ManageIQ/manageiq/blob/master/app/controllers/application_controller.rb#L719